test/e2e/rebuild/basic_rebuild_test.go

```diff
@@ -0,0 +1,99 @@
+package basic_rebuild_test
+
+import (
+	"e2e-basic/common"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"time"
+)
+
+var (
+	pvcName      = "rebuild-test-pvc"
+	storageClass = "mayastor-nvmf"
+)
+
+func basicRebuildTest() {
+	pvc, err := common.GetPVC(pvcName)
+	Expect(err).To(BeNil())
+	Expect(pvc).ToNot(BeNil())
+
+	uuid := string(pvc.ObjectMeta.UID)
+	repl, err := common.GetNumReplicas(uuid)
+	Expect(err).To(BeNil())
+	Expect(repl).Should(BeEquivalentTo(1))
+
+	// Add another child which should kick off a rebuild.
+	common.UpdateNumReplicas(uuid, 2)
+	repl, err = common.GetNumReplicas(uuid)
+	Expect(err).To(BeNil())
+	Expect(repl).Should(BeEquivalentTo(2))
+
+	timeout := "90s"
+	pollPeriod := "1s"
+
+	// Wait for the added child to show up.
+	Eventually(func() int { return common.GetNumChildren(uuid) }, timeout, pollPeriod).Should(BeEquivalentTo(2))
+
+	getChildrenFunc := func(uuid string) []common.NexusChild {
+		children, err := common.GetChildren(uuid)
+		if err != nil {
+			panic("Failed to get children")
+		}
+		Expect(len(children)).Should(BeEquivalentTo(2))
```

Comment:
Could Equal() be used for ints instead of BeEquivalentTo() ?

Reply:
Done, thanks.